examples/step-69/doc/results.dox

```diff
@@ -0,0 +1,151 @@
+
+It is evident that 37k gridpoints for the first-order method is nowhere
+near the resolution needed to resolve any flow features. For comparison,
+here is a "reference" computation with a second-order method and about 9.5M
+gridpoints:
+
+<img src="https://www.dealii.org/images/steps/developer/step-69.2nd-order.t400.jpg" alt="" height="300">
+
+So, we give the first-order method a second chance and run it with about
+2.4M gridpoints on a small compute server:
+
+@verbatim
+# mpirun -np 16 ./step-69 | tee output
+
+[...]
+
+    ####################################################
+    #########                                  #########
+    #########     Cycle  070216  (100.0%)      #########
+    #########      at time t = 3.99999231      #########
+    #########                                  #########
+    ####################################################
+
+TimeLoop<dim>::output(t = 4.00006, checkpoint = 1)
+
++------------------------------------------------------------------------+------------+------------+
+| Total CPU time elapsed since start                                     |  3.34e+05s |            |
+|                                                                        |            |            |
+| Section                                                    | no. calls |  CPU time  | % of total |
++------------------------------------------------------------+-----------+------------+------------+
+| discretization - setup                                     |         1 |      1.96s |         0% |
+| offline_data - assemble lumped mass matrix, and c_ij       |         1 |      10.6s |         0% |
+| offline_data - compute |c_ij|, and n_ij                    |         1 |     0.208s |         0% |
+| offline_data - create sparsity pattern and set up matrices |         1 |      0.38s |         0% |
+| offline_data - distribute dofs                             |         1 |      1.36s |         0% |
+| offline_data - fix slip boundary c_ij                      |         1 |      2.52s |         0% |
+| schlieren_postprocessor - compute schlieren plot           |       201 |      24.3s |         0% |
+| schlieren_postprocessor - prepare scratch space            |         1 |     0.008s |         0% |
+| time_loop - setup scratch space                            |         1 |      1.48s |         0% |
+| time_loop - stalled output                                 |       200 |     0.004s |         0% |
+| time_step - 1 compute d_ij                                 |     70216 |  5.79e+04s |        17% |
+| time_step - 2 compute d_ii, and tau_max                    |     70216 |  5.78e+03s |       1.7% |
+| time_step - 3 perform update                               |     70216 |  2.51e+04s |       7.5% |
+| time_step - 4 fix boundary states                          |     70216 |      56.8s |         0% |
+| time_step - prepare scratch space                          |         1 |     0.028s |         0% |
```

Comment:
Shouldn't these values add up to 100%?

Reply:
I agree. It seems that the percentages are broken for "Total CPU time". I will check.

Reply:
@peterrum The "Total CPU time" accounting was off by a factor of 3. I am not exactly sure why (in particular I haven't seen that on my laptop; we might have fixed something in the library between the two deal.II versions I used). I simply switched the output to wall clock time.